make/generate.mk

```diff
@@ -1,6 +1,7 @@
 .PHONY: generate
 ## generates the asset bundle to be packaged with the binary
 generate:
+	go get github.com/shurcooL/vfsgen
```

Comment:
`github.com/shurcooL/vfsgen` is already declared in `go.mod` and in `pkg/static/assets_generate.go`, so do we need it here as well?

Reply:
no idea why, but the `go mod tidy` in go 1.16 always removes it.

Reply:
ah I see: maybe it's because `pkg/static/assets_generate.go` is not needed to compile the binary? In that case, we could import this package with a blank identifier in `cmd/main.go`:
```
import (
  ...
  _ "github.com/shurcooL/vfsgen"
)
```

Reply:
I have no clue how this works, but I can give it a try. 
And I'm also curious why do we use this and not the go-bindata as in other repositories. @sbryzak ?

Reply:
yeah, it worked 43e46fa
Thanks a lot for the suggestion @xcoulon :pray:

Reply:
or since we're moving to go 1.16, we could also take a look at the `//go:embed` directive: https://golang.org/pkg/embed/

Reply:
but I'm not sure if the `//go:embed` directive will give us all the flexibility we have with go-bindata with regards to switching assets in tests. That's something to look at.